modules/runner/src/integrationTest/java/org/apache/ignite/internal/table/ItInternalTableTest.java

```diff
@@ -384,78 +359,190 @@ public void testRoScanAllExplicitPopulatingData() throws InterruptedException {
         roScanAll(false);
     }
 
-    private void roScanAll(boolean implicit) throws InterruptedException {
-        IgniteImpl node = node();
+    @Test
+    public void getAllOrderTest() throws Exception {
+        var keyRows = populateEvenKeysAndPrepareEntriesToLookup(true);
 
         InternalTable internalTable = ((TableImpl) table).internalTable();
+        var schemaDescriptor = ((TableImpl) table).schemaView().schema();
```

Comment:
It is incorrect to use `var` because the type is not visible from the line of code.

Reply:
This rule does not applicable for tests.

Reply:
This is not indicated here, so it is applicable everywhere in the project.
https://cwiki.apache.org/confluence/display/IGNITE/Java+Code+Style+Guide

Reply:
Ok